src/components/Dashboard/Order/Order.js

```diff
@@ -1,20 +1,67 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAngleDown, faAngleUp, faShippingFast } from '@fortawesome/free-solid-svg-icons';
+import ProductsInOrder from './ProductsInOrder';
+import ConnectedAuthorization from '../../Auth/Authorization';
 
 export default class Order extends Component {
+  state = {
+    details: false,
+  };
+
+  handleClickDetails = () => {
+    this.setState(state => ({ details: !state.details }));
+  };
+
   render() {
+    const { details } = this.state;
     const {
+      handleClick,
+      token,
+      order,
       order: {
         id, person, totalPrice, status, date,
       },
     } = this.props;
     return (
       <React.Fragment>
-        <td>{id}</td>
-        <td>{person}</td>
-        <td>{date}</td>
-        <td>{totalPrice}</td>
-        <td>{status}</td>
+        <tr>
+          <td>{id}</td>
+          <td>{person}</td>
+          <td>{date}</td>
+          <td>{totalPrice}</td>
+          <td>{status}</td>
+          <td>
+            <button type="button" className="btn-orders-detail" onClick={this.handleClickDetails}>
+              {details ? (
+                <FontAwesomeIcon icon={faAngleUp} />
+              ) : (
+                <FontAwesomeIcon icon={faAngleDown} />
+              )}
+            </button>
+          </td>
+          <ConnectedAuthorization
+            render
+            withRoleAdmin={(
+              <td className="orders_realized">
+                <button
+                  className="btn btn-submit"
+                  type="button"
+                  disabled={order.status === 'realized'}
+                  onClick={() => handleClick(order.id, token)}
+                >
+                  <FontAwesomeIcon icon={faShippingFast} />
+                </button>
+              </td>
+)}
```

Comment:
Indentation. Check your editor settings as it should be catched by linter/formatter.